src/borg/archiver.py

```diff
@@ -2474,6 +2474,8 @@ def define_common_options(add_common_option):
             add_common_option('--debug-profile', metavar='FILE', dest='debug_profile', default=None,
                               help='Write execution profile in Borg format into FILE. For local use a Python-'
                                    'compatible file can be generated by suffixing FILE with ".pyprof".')
+            add_common_option('--rsh', metavar='RSH', dest='rsh',
+                              help='additional arguments to pass to the ssh command (default: "ssh")')
```

Comment:
Yeah. Could be clearer. How about:

"Use this command to connect to the remote server (default: 'ssh')"

Reply:
ok.

Reply:
hmm, no. :)

Reply:
"remote" is already saying too much. it's the usual case, but borg@localhost is also quite frequent to avoid permissions mixup when running borg as root.

so "... to the borg serve process" maybe?

Reply:
Some options:

1. "Use this command to connect to the 'borg serve' process (default: 'ssh')"
2. "Use this command to connect to the (remote) borg process (default: 'ssh')"

Reply:
1

Reply:
especially if one tweaks with --rsh, it could rather be non-remote (because for remote, one would use ssh usually).

Reply:
Makes sense. Changed.